.pylintrc

```diff
@@ -398,7 +398,7 @@ valid-metaclass-classmethod-first-arg=mcs
 [DESIGN]
 
 # Maximum number of arguments for function / method
-max-args=6
+max-args=7
```

Comment:
I'd prefer not to change the pylintrc, disabling in the file if you can?

Reply:
Done!